crates/ty_python_semantic/src/program.rs

```diff
@@ -182,9 +200,10 @@ impl PythonVersionSource {
 #[cfg_attr(test, derive(strum_macros::EnumIter))]
 enum PythonSourcePriority {
     Default = 0,
-    PyvenvCfgFile = 1,
-    ConfigFile = 2,
-    Cli = 3,
+    InstallationDirectoryLayout = 1,
+    PyvenvCfgFile = 2,
```

Comment:
I think I still prefer not to have the values assigned. It's so noisy when making updates ;)